Objects/complexobject.c

```diff
@@ -475,31 +475,31 @@ complex_hash(PyComplexObject *v)
 }
 
 /* This macro may return! */
-#define TO_COMPLEX(obj, c) \
-    if (PyComplex_Check(obj)) \
-        c = ((PyComplexObject *)(obj))->cval; \
-    else if (to_complex(&(obj), &(c)) < 0) \
+#define TO_COMPLEX(obj, c)                      \
+    if (PyComplex_Check(obj))                   \
+        c = ((PyComplexObject *)(obj))->cval;   \
+    else if (real_to_complex(&(obj), &(c)) < 0) \
         return (obj)
 
 static int
-to_float(PyObject **pobj, double *dbl)
+real_to_float(PyObject **pobj, double *dbl)
```

Comment:
I think that it is better to use `real_to_double`, because the result is a C double, not a C or Python float.

Reply:
Then maybe -> real_to_Py_complex too?